src/main/java/frc/robot/commands/DropNoteCommand.java

```diff
@@ -38,7 +37,6 @@ public boolean isFinished() {
     @Override
     public void end(boolean interrupted) {
         scoring.stopRoller();
-        scoring.stopLoader();
         timer.stop();
         timer.reset();
```

Comment:
Replace with timer.reset().

Reply:
timer.reset() only sets the timer to 0 it doesnt stop the timer

Reply:
Oh sorry, I meant timer.restart(), but I saw your previous comment about this method.